README.md

```diff
@@ -1 +1,26 @@
-# Week3-Khan-ElBash
\ No newline at end of file
+# Week3-Khan-ElBash
+
+Khan El-Bash's week 3 app.
+
+## Outline
+Our app will:
+- let the user enter a search term
+- use the Online Movie Database (OMDb) API to display information about the film most relevant to the search term
```

Comment:
Does the first choice necessarily mean it is the most relevant? I don't think so.